contrib/colab/DC-GAN.ipynb

```diff
@@ -65,41 +39,41 @@
     },
     {
       "cell_type": "code",
+      "execution_count": null,
       "metadata": {
-        "id": "OApBOAe1fpH_",
+        "colab": {},
         "colab_type": "code",
-        "colab": {}
+        "id": "OApBOAe1fpH_"
       },
+      "outputs": [],
       "source": [
-        "!pip install cloud-tpu-client==0.10 torch==1.9.0 https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.9-cp37-cp37m-linux_x86_64.whl"
-      ],
-      "execution_count": null,
-      "outputs": []
+        "!pip install cloud-tpu-client==0.10 torch==1.9.0 https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.11-cp37-cp37m-linux_x86_64.whl"
```

Comment:
opening the colab file as colab file adds extra changes to the file, it would be cleaner to open in vim to do the changes.

torch==1.11.0 here. it wasn't changed to 1.10 because we didn't have a custom wheel for colab, for 1.11, we do.